tests/docker/eos-storage/Dockerfile

```diff
@@ -3,8 +3,13 @@ FROM gitlab-registry.cern.ch/dss/eos/eos-ci:5.1.25
 COPY scripts/eos-run.sh /mnt/scripts/eos-run.sh
 COPY sssd/sssd.conf /etc/sssd/sssd.conf
 
-RUN ulimit -n 1024000 && yum install -y sssd sssd-client
+# RUN ulimit -n 1024000 && yum install -y sssd sssd-client
+# needs to be run from docker like this:
+# docker run myeoscontainer --ulimit nofiles:1024000:1024000
+# or in the ulimits directive for docker compose
+
```

Comment:
```suggestion
```

Reply:
(do we need that?)

Reply:
I've kept the documentation as is needed to debug issues.